tests/AbstractEndpointTest.php

```diff
@@ -27,6 +27,8 @@ abstract class AbstractEndpointTest extends WebTestCase
     use FixturesTrait;
     use GetUrlTrait;
 
+    protected $apiVersion = 'v2';
```

Comment:
This should probably be a constant, but I'm not fussed either way.

Reply:
should be overridable, hence not a constant. see V1 api tests.